app/components/Channels/ChannelSummaryListItem.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Box, Flex } from 'rebass'
+import { Card, Heading, Text } from 'components/UI'
+import { ChannelCapacity, ChannelMoreButton, ChannelStatus } from 'components/Channels'
+import withEllipsis from 'components/withEllipsis'
+
+const ClippedHeading = withEllipsis(Heading.h1)
+const ClippedText = withEllipsis(Text)
+
+const ChannelSummaryListItem = ({
+  channelId,
+  channelName,
+  channelPubKey,
+  cryptoCurrency,
+  cryptoCurrencies,
+  localBalance,
+  remoteBalance,
+  status,
+  setCryptoCurrency,
+  showChannelDetail,
+  ...rest
+}) => (
+  <Card {...rest}>
+    <Flex>
+      <Box width={8 / 20}>
+        <ChannelStatus status={status} />
+        <ClippedHeading my={1}>{channelName || channelPubKey.substring(0, 10)}</ClippedHeading>
```

Comment:
Actually, I had reworked this already in a later commit but didn't include in this PR as it's outside of the scope of what I'm doing here.

The approach that I have taken is to inject a new `pubKeyShort` prop into the channel object in the reducers/selectors so that we already have a prop available for this and can just use it directly.

I've updated the component in this PR to use a dedicated prop for the short pubkey rather than truncating within the component render method.